File: app/src/main/java/com/example/pijus/chemijosprojektas/PossibleAmount/avizos2.java
package com.example.pijus.chemijosprojektas.PossibleAmount;

import android.os.Bundle;

import com.example.pijus.chemijosprojektas.R;


public class avizos2 extends BaseCalculatePortionsActivity {
    double dribsniai = 6;
    double miltai = 5;
    double vendens = 1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_avizos2);
        calculateButton = findViewById(R.id.buttona3);
        editTexts.add(findViewById(R.id.ivestisa1));
        editTexts.add(findViewById(R.id.ivestisa2));
        editTexts.add(findViewById(R.id.ivestisa3));
        ingredients.add(dribsniai);
        ingredients.add(miltai);
        ingredients.add(vendens);
        answer = findViewById(R.id.textViewa2);
        calculateButton.setOnClickListener(v ->
                calculatePortion()
        );
    }
}

File: app/src/main/java/com/example/pijus/chemijosprojektas/PossibleAmount/avizos2.kt
package com.example.pijus.chemijosprojektas.PossibleAmount

import android.os.Bundle
import com.example.pijus.chemijosprojektas.R

open class avizos2 : BaseCalculatePortionsActivity() {
    var dribsniai = 6.0
    var miltai = 5.0
    var vendens = 1.0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_avizos2)
        calculateButton = findViewById(R.id.buttona3)
        editTexts.add(findViewById(R.id.ivestisa1))
        editTexts.add(findViewById(R.id.ivestisa2))
        editTexts.add(findViewById(R.id.ivestisa3))
        ingredients.add(dribsniai)
        ingredients.add(miltai)
        ingredients.add(vendens)
        answer = findViewById(R.id.textViewa2)
        calculateButton!!.setOnClickListener { calculatePortion() }
    }
}